types/office-js-preview/index.d.ts

```diff
@@ -93579,6 +93579,17 @@ declare namespace Word {
          * @param documentCompareOptions Optional. The additional options that specifies the behavior of comparing document.
          */
         compare(filePath: string, documentCompareOptions?: Word.DocumentCompareOptions): void;
+        /**
+         * Displays revision marks that indicate where the specified document differs from another document.
+         *
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param base64File Required. The Base64-encoded content of the document with which the specified document is compared.
+         * @param documentCompareOptions Optional. The additional options that specifies the behavior of comparing document. Note that compareTarget isn't allowed to be CompareTargetSelected in this API.
```

Comment:
The second sentence doesn't mean much of out context. Is there a way to rephrase that for clarity?
```suggestion
         * @param documentCompareOptions Optional. The additional options that specify the behavior of comparing document. Note that `compareTarget` isn't allowed to be `CompareTargetSelected` in this API.
```